core-galleon-pack/pom.xml

```diff
@@ -93,6 +93,19 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
```

Comment:
All  the added dependencies should have a provided scope.

Reply:
Fixed, thanks!